scripts/accumulator-server3.py

```diff
@@ -0,0 +1,592 @@
+#!/usr/bin/env python3
```

Comment:
I'd suggest to keep only one version of the script. Could you rename it to scripts/accumulator-server.py replacing the old one, please?

Reply:
I think now it's done :wink:

Reply:
Fixed in 39b55b6fedb02f784a4f64208e22097646f985d7